bin/improver-tests

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#------------------------------------------------------------------------------
+# (C) British Crown Copyright 2017 Met Office.
+#------------------------------------------------------------------------------
+# NAME
+#    improver tests - Run IMPROVER self-tests
+#
+# SYNOPSIS
+#    improver tests
+#
+# DESCRIPTION
+#    Launch all IMPROVER self-tests.
+#------------------------------------------------------------------------------
+
+set -eu
+
+function echo_ok {
+    echo -e "\033[1;32m[OK]\033[0m $1"
+}
+
+cd $IMPROVER_DIR/lib
+
+# PEP8 testing.
+${PEP8:-pep8} improver
+echo_ok "pep8"
+
+# Pylint obvious-errors-only testing.
+${PYLINT:-pylint} -E --rcfile=../etc/pylintrc improver
```

Comment:
Would it be possible to have the output from pylint printed in someway? So we can see the scores and where there are problems. it prints out the output from the unit tests for us.